rosidl_typesupport_cpp/test/test_cli_extension.py

```diff
@@ -14,15 +14,19 @@
 
 import pathlib
 
+from ament_index_python import get_resources
 from rosidl_cli.command.generate.api import generate
 
 TEST_DIR = str(pathlib.Path(__file__).parent)
 
 
 def test_cli_extension_for_smoke(tmp_path):
+    ts = 'cpp[typesupport_implementations:{}]'.format(
+        list(get_resources('rosidl_typesupport_cpp'))
+    )
     generate(
         package_name='rosidl_typesupport_cpp',
         interface_files=[TEST_DIR + ':msg/Test.msg'],
-        typesupports=['cpp'],
```

Comment:
Same comment about having two tests, one with specs and one without.

Reply:
See https://github.com/ros2/rosidl_typesupport/pull/112/commits/2e7e8106b22e699ee8704e54b098b4d6ecf9b860.